src/components/PackageCard/PackageCard.scss

```diff
@@ -18,6 +18,10 @@
   flex-grow: 1;
 }
 
+.sc-package-card__body p:hover{
```

Comment:
I'd change this to `.sc-package-card__body :hover` so it applies to any other elements

Reply:
It's just that other cards' heading tags were underlined. I'll make a change!